src/KickMCMC.jl

```diff
@@ -169,235 +291,160 @@ function createEccentricMCMCModel(observations::Vector{Symbol}, observed_values:
+        v_r_100kms ~ vsys_r_dist
+        v_N = v_N_100kms*100*km_per_s
+        v_E = v_E_100kms*100*km_per_s
+        v_r = v_r_100kms*100*km_per_s
 
         #m1 is assumed to remain constant, no impact velocity
-        a_f, e_f, Ω_f, ω_f, i_f, v_n, v_e, v_rad = 
+        a_f, e_f, Ω_f, ω_f, i_f, v_N, v_E, v_r = 
             post_supernova_general_orbit_parameters(m1=m1, m2=m2, a=a, e=e, m2_f=m2_f, 
-                vkick=vkick*100*km_per_s, θ=θ, ϕ=ϕ, ν=ν, Ω=Ω, ω=ω, i=i)
+                vkick=vkick, θ=θ, ϕ=ϕ, ν=ν, Ω=Ω, ω=ω, i=i)
         P_f = kepler_P_from_a(m1=m1, m2=m2_f, a=a_f)
         K1 = RV_semiamplitude_K1(m1=m1, m2=m2_f, P=P_f, e=e_f, i=i_f)
         K2 = RV_semiamplitude_K1(m1=m2_f, m2=m1, P=P_f, e=e_f, i=i_f)
 
-        for i in eachindex(obs)
-            obs_symbol = obs[i]
+        for i in eachindex(observations.props)
+            obs_symbol = observations.props[i]
             if obs_symbol == :P
-                obs_vals[i] ~ Cauchy(P_f, obs_errs[i])
+                param = P_f
             elseif obs_symbol == :e
-                obs_vals[i] ~ Cauchy(e_f, obs_errs[i])
+                param = e_f
             elseif obs_symbol == :K1
-                obs_vals[i] ~ Cauchy(K1, obs_errs[i])
+                param = K1
             elseif obs_symbol == :K2
-                obs_vals[i] ~ Cauchy(K2, obs_errs[i])
+                param = K2
             elseif obs_symbol == :m1
-                obs_vals[i] ~ Cauchy(m1, obs_errs[i])
+                param = m1
             elseif obs_symbol == :m2
-                obs_vals[i] ~ Cauchy(m2_f, obs_errs[i])
+                param = m2_f
+
+                # RTW TODO: fix these dists
             elseif obs_symbol == :Ω
                 #obs_vals[i] ~ VonMises(Ω_f, 1/obs_errs[i]^2)#Normal(Ω_f, obs_errs[i])
-                obs_vals[i] ~ WrappedCauchy(Ω_f, obs_errs[i])
+                obs_vals[i] ~ angular_likelihood_dist(Ω_f, obs_errs[i])
             elseif obs_symbol == :ω
                 #obs_vals[i] ~ VonMises(ω_f, 1/obs_errs[i]^2)#Normal(ω_f, obs_errs[i])
-                obs_vals[i] ~ WrappedCauchy(ω_f, obs_errs[i])
+                obs_vals[i] ~ angular_likelihood_dist(ω_f, obs_errs[i])
             elseif obs_symbol == :i
-                obs_vals[i] ~ Cauchy(i_f, obs_errs[i])
+                obs_vals[i] ~ likelihood_dist(i_f, obs_errs[i])
+                # RTW: what is going on with these, v_N - v_N?
             elseif obs_symbol == :v_N
-                obs_vals[i] ~ Cauchy(v_N - v_N, obs_errs[i])
+                param = v_N - v_N
             elseif obs_symbol == :v_E
-                obs_vals[i] ~ Cauchy(v_E - v_E, obs_errs[i])
+                param = v_E - v_E
             elseif obs_symbol == :v_r
-                obs_vals[i] ~ Cauchy(v_r - v_r, obs_errs[i])
+                param = v_r - v_r
             end
+            likelihood == :Cauchy ?
+              obs_vals[i] ~ Cauchy(param, obs_errs[i]) :
+              obs_vals[i] ~ Normal(param, obs_errs[i]) 
```

Comment:
In here we need to adjust things a bit. If the variable is an angular variable we need to use WrappedCauchy of Von Mises. Otherwise we use Cauchy or Normal